vhost_user_net/src/lib.rs

```diff
@@ -151,42 +143,193 @@ impl VhostUserNetBackend {
         Self::new_with_tap(taps, num_queues, queue_size)
     }
 
+    pub fn set_vring_worker(&mut self, vring_worker: Option<Vec<Arc<VringWorker>>>) {
```

Comment:
Looks like you moved some code around on this commit, which makes the review harder. Could you make sure you only make the changes needed, and do the moving into a different commit?

Reply:
Oh, I'm sorry for making review hard. Since NetEpollHandler is added in vhost_user_net backend, I re-adjust the code layout, some functions are moved around. There are implementations for the three parts currently: VhostUserNetBackend, VhostUserBackend  and NetEpollHandler. For all the changes in this commit aim to enable NetEpollHandler actually, I think we could keep them as is.